src/main/java/com/github/bartimaeusnek/cropspp/crops/TC/ShimmerleafCrop.java

```diff
@@ -29,14 +29,8 @@ public String name() {
 
     @Override
     public int growthDuration(ICropTile crop) {
-        int r = 9999999;
-        if (ConfigValues.debug)
-            r = 1;
-        else if (crop.getSize() == 1)
-            r = 2250;
-        else if (crop.getSize() == 2 && (crop.isBlockBelow("blockQuicksilver") || !OreDictionary.doesOreNameExist("blockQuicksilver")))
-            r = 1750;
-        return r;
+        if (ConfigValues.debug) return 1;
+        return crop.getSize() == 1 ? 2250 : 1750;
```

Comment:
Missing check for Quicksilver block below. Intended?

Reply:
Intended, handled by canGrow()